frontend/src/main/components/BasicCourseSearch/CourseOverTimeSearchForm.js

```diff
@@ -61,7 +61,7 @@ const CourseOverTimeSearchForm = ({ fetchJSON }) => {
     } else {
         setCourseSuf("");
     }
-};
+  };
 
 
   // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
This comments needs to be removed; it's too broad an exception, plus we know how to test for this
(see: https://ucsb-cs156.github.io/topics/stryker/stryker_html_attribute_surviving_mutants.html)

Reply:
Please see the three most recent commits below. Is this what you were expecting me to add? Or am I still missing something (in regards to the stryker tests)?